lib/taste_tester/client.rb

```diff
@@ -125,14 +127,93 @@ def upload
 
     private
 
+    def populate(stream, writer, path, destination)
+      full_path = File.join(File.join(TasteTester::Config.repo, path))
+      return unless File.directory?(full_path)
+      # everything is relative to the repo dir. chdir makes handling all the
+      # paths within this simpler
+      Dir.chdir(full_path)
+      Find.find('.') do |p|
+        next if p == '.'
+        name = File.join(destination, p[2..-1])
+        if File.directory?(p)
+          # skip it. This also handles symlinks to directories which aren't
+          # useful either.
+        elsif File.symlink?(p)
+          # tar handling of filenames > 100 characters gets complex. We'd use
+          # split_name from Minitar, but it's a private method. It's
+          # reasonable to assume that all symlink names in the bundle are
+          # less than 100 characters long. Long term, the version of minitar
```

Comment:
check if chefdk3 or 4 has a newer one (fb is necessarily on chefdk2)... if not, then file a request upstream.

Reply:
I looked closer: the newest minitar doesn't explicitly support symlinks either. I think chasing a dependency on a dependency isn't particularly urgent. I'll leave the comment to explain where this might be improved. It's just not urgent at all.